didcomm/did_doc/did_resolver.py

```diff
@@ -0,0 +1,56 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional
+
+from didcomm.did_doc.did_doc import DIDDoc
+
+
+class DIDResolver(ABC):
```

Comment:
Sometimes we need to resolve KID for external DID. How we plan to do this? It would be nice to avid DID parsing in DID Comm.